dockstore-client/src/main/java/io/dockstore/client/cli/CheckerClient.java

```diff
@@ -366,23 +366,19 @@ private void testParameterChecker(List<String> args) {
 
     private void updateVersionChecker(List<String> args) {
         if (containsHelpRequest(args) || args.isEmpty()) {
-            versionTagHelp();
+            updateVersionCheckerHelp();
         } else {
             // Retrieve arguments
             String entryPath = reqVal(args, ENTRY);
-
             // Get entry from path
             Entry entry = getEntryFromPath(entryPath);
-
             // Get checker workflow
             Workflow checkerWorkflow = getCheckerWorkflowFromEntry(entry, true);
-
             // Update version
             if (entry != null && checkerWorkflow != null) {
-                // Readd entry path to call, but with checker workflow
```

Comment:
In honor of this change, I shall now queue up some tunes from my favorite '80s-hair-metal-band-with-a-similarly-misspelled-name (Ratt).